scipy/optimize/_qap.py

```diff
@@ -120,8 +120,8 @@ def quadratic_assignment(A, B, method="faq", options=None):
     ...               [0, 0, 0, 3], [0, 0, 0, 0]])
     >>> res = quadratic_assignment(A, B)
     >>> print(res)
-     col_ind: array([0, 3, 2, 1])
          fun: 3260
+     col_ind: [0 3 2 1]
```

Comment:
removing `array` from the output seems odd, this makes it look like a list. It only saves 7 characters to do that, so is this really a good idea?

Reply:
As we discussed, this is how NumPy implements `__str__`:
```
import numpy as np
print(np.array([1, 2, 3]))  # [1 2 3]
```
True, this is the `__repr__` method of `OptimizeResult`, but it has always behaved more like a `__str__` method than a `__repr__`, so this is just making that more consistent.

Reply:
Thanks for the explanation @mdhaber. If there is a consistent logic here - `__repr__` instead of `__str__ - that's enough of a reason for me. I'll resolve this.